xapian-core/cluster/cluster.cc

```diff
@@ -252,15 +252,12 @@ Point::initialize(TermListGroup &tlg, const Document &doc_) {
     }
 }
 
-void
-Centroid::set_to_point(Point &p) {
-    LOGCALL_VOID(API, "Centroid::set_to_point()", p);
-    magnitude = 0;
+Centroid::Centroid(Point &p) {
     for (TermIterator it = p.termlist_begin(); it != p.termlist_end(); ++it) {
 	termlist.push_back(Wdf(*it, 1));
```

Comment:
Why are we storing the `Wdf` objects, if we ignore the wdf part of them when making centroids out of points? Do we actually use them at all, or do we only ever use them from `values`?